### PR TITLE
[1LP][RFR] New default tenant deletion test case and table xpath correction

### PR DESCRIPTION
**Pull request by vatsalparekh**
Purpose or Intent
=================
{{ pytest: -vvvv cfme/tests/configure/test_access_control.py -k test_delete_default_tenant }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7828c92a2d5b3b150bc69c6a9c8d1b664cf35e22:

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 574:13: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6e791f04ade018898475f4f8418563b2eef2a53d:

`cfme/configure/access_control.py`:
- :red_circle: Line 21:1: **F401** *'cfme.utils.version' imported but unused*
- :bangbang: Line 1265:101: **E501** *line too long (119 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -1261,7 +1259,9 @@ def is_displayed(self):
 class AllTenantView(ConfigurationView):
     """ All Tenants View """
     toolbar = View.nested(AccessControlToolbar)
-    table = Table('//*[@id="fieldset"]/table')
+    table = Table(VersionPick(
+        {Version.lowest(): '//*[@id="records_div"]/table',
```
Comment (RonnyPfannschmidt) [inline cfme/configure/access_control.py]: **REQUIRED** why the change from `//*[@id="fieldset"]/table` to `//*[@id="records_div"]/table`

Comment (vatsalparekh) [reply cfme/configure/access_control.py]: Its the updated xpath, there were no tests using this table, so the incorrect xpath was not noticed
